NLibrary/Classes/PlaylistMVVM/PlaylistModel.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  UserProtocol.swift
+//  taylorswift
+//
+//  Created by Nathan Ngobale on 2020/03/20.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import Foundation
+import AVKit
+
+public struct PlaylistBasicModel {
+//    public var playlistItems: [PlaylistModel] = []
+//    public var authorId: String?
+//    init?(dict: NSDictionary) {
+//        guard let keyValues = dict.allValues.first as? NSDictionary,
+//            let authorId = dict.allKeys.first as? String else {
+//            return
+//        }
+//        self.authorId = authorId
+//        for (key, value) in keyValues {
+//            guard let key = key as? String,
+//                let page = value as? NSDictionary else {
+//                return
+//            }
+//            playlistItems.append(PlaylistModel(albumName: key, dictionary: page))
+//        }
+//    }
+//    var dict: [String: Any]?
+}
+
```

Comment:
Maybe just remove the commented code if it's not being used.

Reply:
Thanks will do so now